src/Symfony/Component/Config/Resource/ComposerResource.php

```diff
@@ -71,12 +71,14 @@ private static function refresh()
     {
         self::$runtimeVendors = [];
 
+        $i = 0;
         foreach (get_declared_classes() as $class) {
             if ('C' === $class[0] && 0 === strpos($class, 'ComposerAutoloaderInit')) {
                 $r = new \ReflectionClass($class);
                 $v = \dirname(\dirname($r->getFileName()));
                 if (file_exists($v.'/composer/installed.json')) {
-                    self::$runtimeVendors[$v] = @filemtime($v.'/composer/installed.json');
+                    self::$runtimeVendors["__vendor_$i"] = @filemtime($v.'/composer/installed.json');
```

Comment:
This break the getVendors method currently.

Reply:
Better ?